i18n/da.toml

```diff
@@ -106,7 +106,7 @@ battery = "Batteri"
 [main.mode]
 off = "Af"
 minpv = "Min+Solar"
-pv = "Solar"
+pv = "Solenergi"
```

Comment:
Ich vermute diese Änderungen sind nicht gewollt hier im PR? Magst du mal schauen wo das herkommt?

Reply:
Die ist nicht von mir. Ich hatte vermutet, dass die durch einen Merge vom master Branch rein gekommen ist.

Reply:
Ja, das dachte ich mir. Magst du schauen ob du das wieder zurückgedreht bekommst? Im Zweifelsfall einmal die Master-TOML Dateien in deinen Branch kopieren.

Reply:
git rebase

Kopieren ist egtl. immer eine schlechte Idee.